src/main/java/org/example/DeathMinigames/commands/GameCMD.java

```diff
@@ -30,12 +32,8 @@ public boolean canUse(@NotNull CommandSender sender) {
     public void execute(CommandSourceStack stack, String[] args) {
         Player player = (Player) stack.getSender();
         if (args.length == 1) {
-            if (args[0].equalsIgnoreCase("difficulty")) {
-                player.sendMessage(Component.text("Deine Schwierigkeit ist gerade bei ").color(NamedTextColor.GOLD)
-                        .append(Component.text(Difficulty.getDifficulty(player)).color(NamedTextColor.RED)));
-            }
-            else {
-                if(args[0].equalsIgnoreCase("lowerDifficulty")) {
+            switch (args[0]) {
```

Comment:
When using command arguments they should be forced to lower case or be set as final Strings
```suggestion
            switch (args[0].to lower case()) {
```